api/src/pcapi/core/bookings/validation.py

```diff
@@ -125,23 +124,15 @@ def check_is_usable(booking: Booking) -> None:
     is_booking_for_event_and_not_confirmed = booking.stock.beginningDatetime and not booking.isConfirmed
     if is_booking_for_event_and_not_confirmed:
         venue_departement_code = booking.venue.departementCode
-        booking_date = datetime.datetime.strftime(
-            utc_datetime_to_department_timezone(booking.dateCreated, venue_departement_code), "%d/%m/%Y à %H:%M"
-        )
         max_cancellation_date = datetime.datetime.strftime(
             utc_datetime_to_department_timezone(
-                api.compute_cancellation_limit_date(
-                    booking.stock.beginningDatetime,
-                    booking.dateCreated,
-                ),
+                booking.cancellationLimitDate,
```

Comment:
Programmation défensive : peut-être faire un 
```
if not booking.cancellationLimitDate or booking.cancellationLimitDate < datetime.utcnow():
   logger.exception("cas chelou mais rien ne l'interdit niveau typage")
   raise BookingIsNotConfirmed("Vous pourrez valider cette contremarque une fois le délai d’annulation passé")
```

Reply:
C'est à peu près ce que fait booking.isConfirmed : `return self.cancellationLimitDate is not None and self.cancellationLimitDate <= datetime.utcnow()`
On ne peut pas être ici sans cancellationLimitDate ou avec une cancellationLimitDate < now()